Make.config

```diff
@@ -508,12 +508,12 @@ DOTNET_FEED_DIR ?= $(DOTNET_DESTDIR)/nuget-feed
 # avoid consuming a lot of disk space (since they're never automatically deleted). The system-dependencies.sh script will install locally as long
 # as there's a TARBALL url.
 
+-include $(TOP)/net6.config
```

Comment:
Running msbuild to generate this file seems a bit heavy, when a simple make target can do it (in the top-level Makefile):

```make
net6.config: env/Versions.props
    grep MicrosoftDotnetSdkInternalPackageVersion env/Versions.props | sed "some expression here" > $@.tmp
    mv $@.tmp $@
```